### ISSUE TITLE
`SOLCVER` detection process breaks when using globs

### ISSUE THREAD

**Issue by RatanRSur** [crytic/slither-action#2]
Locally, slither works fine with globs as crytic-compile unglobs them but with this action it greps for the `target` to find the solidity version. This seems to break when you use globs.

**Comment by elopez** [comment]
Thanks for the report! You make a good point. It probably doesn't make much sense to try to guess if the target is a directory 🤔  in that case the compilation framework should take care of it already.

This problem goes away if the user passes `solc-version`, as guessing gets skipped then. If they are targeting a single `.sol` file, they should probably be indicating a specific version instead of guessing.

**Comment by elopez** [comment]
@RatanRSur does the fix in #15 cover the use case you had in mind? I did a few tests with globs (example below) and it seems to behave as expected now:

```yaml
name: Slither Analysis
on: [push]
jobs:
  analyze:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
      - uses: crytic/slither-action@dev-glob-target
        with:
          target: '**/*.sol'
```

**Comment by RatanRSur** [comment]
I've moved past this and don't have the time to verify but I'll optimistically close this and reopen if it is an issue :) Thanks for fixing!
